.github/workflows/validate-new-news.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Validate new news entries
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: Get out of detached head state
+        run: |
+          git fetch origin ${{ github.base_ref }}
+          git fetch origin ${{ github.ref }}
+          git checkout FETCH_HEAD --
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Validate new news items
+        run: |
+          git diff --name-status origin/${{ github.base_ref }}
+          git diff --name-status origin/${{ github.base_ref }}\
+          | python scripts/verify-news-fragments.py
```

Comment:
Doesn't the script work without the diffs now?

Reply:
Ughh, well remembered...